strings/camel_case_to_snake_case.py

```diff
@@ -0,0 +1,83 @@
+"""
+this function focuses on converting camel case (eg: camelCase)
```

Comment:
Typically you talk about a function within a function and talk about what the file does outside of a function. Here you could explain what snake_case and camelCase is, where they're used, provide documentation, etc